scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -18,7 +18,6 @@ local GAME_TIE = 3;
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
     if (npcUtil.tradeHas(trade, nil, 5)) then
```

Comment:
That tradeHas check should also verify that the player didn't give him an item.

Although I'll note that DSP's current npcUtil.tradeHas is broken (in regards to items at least). I fixed it while working on Adoulin NPCs, but haven't PRed it yet (intend to do so soon, after my PR for log tables gets merged).